capture.go

```diff
@@ -0,0 +1,140 @@
+// Package command provides an organized way of running a *exec.Cmd and generates
+// a Capture that contains all commonly-accessed information in the Cmd that allows
+// for repeated execution.
+package command
+
+import (
+	"context"
+	"errors"
+	"os/exec"
+)
+
+// Capture represents the state for a run of a command.
+type Capture struct {
+	// Name is the name to run.
+	//
+	// This is the only field that must be set to a non-zero
+	// value. If Path is relative, it is evaluated relative
+	// to Dir.
+	Name string `json:"name"`
+
+	// Args holds name line arguments, excluding the name as Args[0].
+	// This differs in behavior from the exec.Cmd type.
+	Args []string `json:"args,omitempty"`
+
+	// Dir specifies the working directory of the name.
+	// If Dir is the empty string, Comand runs the name in the
+	// calling process's current directory.
+	Dir string `json:"working_dir,omitempty"`
+
+	// Env specifies the environment of the process.
+	// Each entry is of the form "key=value".
+	// If Env is nil, the new process uses the current process's
+	// environment.
+	// If Env contains duplicate environment keys, only the last
+	// value in the slice for each duplicate key is used.
+	// As a special case on Windows, SYSTEMROOT is always added if
+	// missing and not explicitly set to the empty string.
+	Env []string `json:"env,omitempty"`
+
+	// Output provides the combined output from the name as a string.
+	Output string `json:"output,omitempty"`
+
+	// ExitCode holds the exit code returned by the call.
+	// It will be 0 (default value) even if a name didn't run due to error.
+	// You MUST check error when calling any of the functions below, as the
+	// exec.ExitError type contains additional context for failure.
+	ExitCode int `json:"exitCode"`
```

Comment:
whats the reason for camelcasing exitCode? we've (metrum) definitely more leaned towards snake_case but also I don't really have a strong preference to explicitly force it. Though I would advocate for consistency, and I see Dir above is working_dir, so I would think we would do either workingDir and exitCode or working_dir and exit_code

Reply:
accident, I'll snake it.